cdk/script/lint

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+yarn lint
```

Comment:
We have a number of scripts like this one (`setup`, `lint`, `start`, `test`). I'm slightly unsure as to if we want these - it's not really how we've done things else where in this project

Reply:
(we do want the `ci` script though as that's doing some stuff!)